static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js

```diff
@@ -108,7 +110,16 @@ define([
 
     function insertAdAtPara(para, name, type) {
         var ad = createSlot(name, type);
-        para.parentNode.insertBefore(ad, para);
+        if (detect.isBreakpoint({max: 'tablet'})) {
+            // Insert ad using steady page
+            // To avoid jumping the user
+            steadyPage.insert(ad, function() {
```

Comment:
@regiskuckaertz Thinking about this more, I think I'm messing up the use of space-filler to insert the ad - i.e: Spacefiller handles the queuing of element insertion, then uses fastdom write to insert which calls this, which then does the read, write part. Spacefiller specifically states not to use fastdom for the writer, which is precisely what I'm doing. Maybe the utility needs to play nicer with spacefiller to avoid jumping animation frames. Thoughts on the approach here?